docs/source/visualize/logging_advanced.rst

```diff
@@ -46,26 +46,6 @@ To change this behaviour, set the *log_every_n_steps* :class:`~pytorch_lightning
 
 ----
 
-Modify flushing frequency
```

Comment:
I wonder, should we keep this and change the code to TensorBoard instead (with its flushing argument)?

Reply:
Should the change be part of this PR? If you think I can include it maybe.

Reply:
Yes if it should if we decide to do it. What do others think? @rohitgr7 @carmocca @daniellepintz

Reply:
yeah! I think that'd be useful

Reply:
@rohitgr7 @awaelchli A bit confused here, should we change the code to include this argument in TensorBoard, or is the implementation present and we use the TensorBoard example here?

Reply:
Can't push to your branch. Here is what I wrote:


```
Modify flushing frequency
=========================

Some loggers keep logged metrics in memory for N steps and only periodically flush them to disk to improve training efficiency.
Every logger handles this a bit differently. For example, here is how to fine-tune flushing for the TensorBoard logger:

.. code-block:: python

    # Default used by TensorBoard: Write to disk after 10 logging events or every two minutes
    logger = TensorBoardLogger(..., max_queue=10, flush_secs=120)

    # Faster training, more memory used
    logger = TensorBoardLogger(..., max_queue=100)

    # Slower training, less memory used
    logger = TensorBoardLogger(..., max_queue=1)


----
```

Would you mind adding that please?

Reply:
Updated!

Reply:
good stuff